src/twisted/test/test_twistd.py

```diff
@@ -865,6 +901,91 @@ def fakeSetupEnvironment(self, chroot, rundir, nodaemon, umask,
             ['/foo/chroot', '/foo/rundir', True, 56, '/foo/pidfile'])
 
 
+    def test_shedPrivileges(self):
+        """
```

Comment:
Empty docstring - maybe something like `L{UnixApplicationRunner.shedPrivileges} switches the process' user ID.`

Reply:
Fixed